FIRO_TSEnsembles/src/main/java/hec/stats/NDayMultiComputable.java

```diff
@@ -16,7 +20,7 @@ public NDayMultiComputable(MultiComputable stepOne, int numberDays) {
     public float compute(float[] values) {
         values = _stepOne.MultiCompute(values);
         int timestep = (int) _c.getDuration().toHours();
-        int timestepDay = timestep / 24;
+        int timestepDay = timestep * 24;
```

Comment:
i think it would be 24/timestep - 24/3 => 8 timesteps make up a day  24/12 => 2 timesteps make up a day... 24/48 => .5 timesteps make up a day.